internal/mock_client/eth_client.go

```diff
@@ -0,0 +1,81 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: github.com/base-org/pessimism/internal/client (interfaces: EthClientInterface)
+
+// Package mock_client is a generated GoMock package.
```

Comment:
Any chance we could instead store this mock file in an `internal/mocks` package that can be used to durably support additional generated mocks without convoluting the internal file structure?

Reply:
+1

Reply:
I moved the file to mocks and updated Makefile, so anytime make test is run, the mock is generated.